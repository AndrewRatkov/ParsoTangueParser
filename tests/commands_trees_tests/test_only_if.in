if
30 != 239
then
int x := 30;
fi 