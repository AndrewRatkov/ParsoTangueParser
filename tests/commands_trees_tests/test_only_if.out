IF
|
+--->[INT,!=]------->[INT,30]
|        |
|        +---------->[INT,239]
|
THEN
|
+--->[DEF int x]--->[INT,30]
|
FI
